src/main/java/net/lepko/easycrafting/core/recipe/RecipeManager.java

```diff
@@ -76,8 +83,13 @@ public static void scanRecipes() {
 		HashMap<String,String> recipeExists = new HashMap<String,String>();
 		for (IRecipe r : recipes) {
 			WrappedRecipe wr = WrappedRecipe.of(r);
-			if (wr != null && !CheckIfRecipeAlreadyExists(allRecipes, wr,recipeExists))
-			{
+			if (wr != null && !CheckIfRecipeAlreadyExists(allRecipes, wr,recipeExists)){
```

Comment:
If i comment these lines out, my search finishes fine. Not sure what the purpose of it is, haven't looked too deep into it.